addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -200,6 +210,12 @@ function factory(dependencies) {
             _newOrdered[index1] = chatWindow2;
             _newOrdered[index2] = chatWindow1;
             this.update({ allOrdered: replace(_newOrdered) });
+            this.update({ _ordered: _newOrdered });
```

Comment:
this LOC should be removed from diff

Reply:
Well I missed that part when I reverted the diff. :(